src/ol/style/expressions.js

```diff
@@ -260,24 +260,35 @@ export function expressionToGlsl(context, value, typeHint) {
     }
     return operator.toGlsl(context, value.slice(1), typeHint);
   }
+
   const valueType = getValueType(value);
   if ((valueType & ValueTypes.NUMBER) > 0) {
     return numberToGlsl(/** @type {number} */ (value));
-  } else if ((valueType & ValueTypes.BOOLEAN) > 0) {
```

Comment:
I've also removed the `else` after `return` to make this function a bit more readable.  I'll follow at some point by proposing that we enforce this with eslint.